ckan/templates/organization/members.html

```diff
@@ -10,7 +10,9 @@
 {% endblock %}
 
 {% block primary_content_inner %}
-  <h3 class="page-heading">{{ _('{0} members'.format(c.members|length)) }}</h3>
+  {% set count = c.members|length %}
+  {% set members_count = _(ungettext('{0} member', '{0} members', count)).format(count) %}
```

Comment:
You shouldn't need to pass the result of `ungettext` to `_`, that would translate it twice. In addition, please use named placeholders in your strings:

    set members_count = ungettext('{count} member', '{count} members', count).format(count=count)